src/map/packet_system.cpp

```diff
@@ -2693,10 +2693,7 @@ void SmallPacket0x058(map_session_data_t* const PSession, CCharEntity* const PCh
 void SmallPacket0x059(map_session_data_t* const PSession, CCharEntity* const PChar, CBasicPacket& data)
 {
     TracyZoneScoped;
-    if (PChar->animation == ANIMATION_SYNTH)
-    {
-        synthutils::sendSynthDone(PChar);
-    }
+    // Do nothing. This is handled in synth state.
```

Comment:
Could we compare the synth state start time if this packet comes in to catch turds?

Reply:
Timings haven't really been dialed in, and this entire state would mean premature synth resuluts would be ignored to begin with. Probably a future problem if someone wants to tackle it